WolvenKit.App/Models/FileSystemModel.cs

```diff
@@ -21,8 +21,20 @@ public class FileSystemModel : INotifyPropertyChanged
     private string _extension = "default";
 
 
+    private static readonly string s_archiveSubfolder =
```

Comment:
This shouldn't be done like this. We already had Functions "IsInRawFolder" and "IsInArchiveFolder" that did this better.